### DISCUSSION TITLE
Is it possible to play audio from widgets?

### DISCUSSION THREAD

**Discussion by Jaskarrr** [baffler/Transparent-Twitch-Chat-Overlay#202] [General]
I've been using StreamElements to conglomerate some widgets, and visually, everything works flawlessly, but having audio from these widgets playing locally on my pc would be a huge boon, because OBS doesn't work for all of my needs. I don't need any routing capability from within the app, it's fine if it just outputs from the default device.

Thank you for your time and this amazing app!

**Comment by baffler** [comment]
I looked into this and I'm able to get it to play the audio. There's a couple more things I need to update so I'll have a release out soon with the fix. 👍

**Comment by Jaskarrr** [comment]
Amazing!!! 
Do you have an estimate on how soon?
Thank you again, you rock!

**Reply by baffler** [reply]
Should be this weekend. I have the other fixes and such in place, just have to test it a bit more. I think it will be either on Saturday or Sunday (October 14th or 15th).
